### PR TITLE
[Large Shuttle] KL Monolith 

### PR DESCRIPTION
**Pull request by CharcoalShipyard**
## About the PR

The Kosmologistika Corporation has developed another ship exclusively for the Frontier Star Sector.
This large ship provides its crew with a wide range of capabilities. At first glance, you can say that everything is there, but don’t worry, because the Kosmologistika Corporation never gives more than necessary. The crew of a Monolith class vessel will be interested in filling empty spaces on the ship, as well as improving its starting condition.

## Why / Balance

Kosmologistika corporation wants to expand its sphere of influence by creating new ships.

## Technical details

Nothing was changed. The files in this PR are just shuttle files.

## Media
monolith
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

No breaking changes here. 

**Changelog**
:cl:
- add: KL Monolith

### CODE REVIEW

**Comment by CharcoalShipyard** [comment]
I don't know how to remove my previous work from the branch I've created.
My last ship was Bulker, and the files for It for some reason are still linkd...

File: Resources/Prototypes/_NF/Shipyard/monolith.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  id: MONOLITH
+  name: KL Monolith
+  description: A large and slow universal ship designed for manual upgrades by the crew.
+  price: 80000 #68k before +15% 
+  category: Large
+  group: Civilian
+  shuttlePath: /Maps/Shuttles/monolith.yml
+
+- type: gameMap
+  id: MONOLITH
```
Comment (Cheackraze) [inline Resources/Prototypes/_NF/Shipyard/monolith.yml:11]: Please use proper naming conventions. You have this all uppercased in random spots and it doesnt match either your own naming conventions and also doesnt match our naming standards. Please do not do this.

File: Resources/Prototypes/_NF/Shipyard/monolith.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+- type: vessel
+  id: MONOLITH
```
Comment (Cheackraze) [inline Resources/Prototypes/_NF/Shipyard/monolith.yml:2]: same here

File: Resources/Prototypes/_NF/Shipyard/monolith.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  group: Civilian
+  shuttlePath: /Maps/Shuttles/monolith.yml
+
+- type: gameMap
+  id: MONOLITH
+  mapName: 'KL Monolith'
+  mapPath: /Maps/Shuttles/monolith.yml
+  minPlayers: 0
+  stations:
+    MONOLITH:
```
Comment (Cheackraze) [inline Resources/Prototypes/_NF/Shipyard/monolith.yml:16]: same here, do not do this. You will also need to change this if you did it on the BecomesStation component.

**Review by Cheackraze — CHANGES_REQUESTED** [review]

**Comment by Cheackraze** [comment]
just bumping this to let you know, still awaiting changes

**Comment by MagnusCrowe** [comment]
Going to close this for now because of inactivity.  Feel free to open it again when you're ready to start working on it again.

**Pull request closed without merging**